team_builder_react/src/components/CheckList.js

```diff
@@ -16,6 +16,8 @@ import Multiselect from 'multiselect-react-dropdown'; // https://www.npmjs.com/p
   https://www.geeksforgeeks.org/how-to-pass-data-from-child-component-to-its-parent-in-reactjs/
 */
 const CheckList = ( { SendToParent, defaultSelected }) => {
+  SendToParent(defaultSelected); // Default list the parent should have
+
   /*
```

Comment:
I am wondering if we should bring that submit button into this component so we can avoid the nauseating amount of SendToParent calls for additions to and ommitions from the skills list.  

Though, it might be more pain than it is worth, because that would effect the layout of your page which displays this component as well.

Reply:
That might be something that we can do in another PR. My main concern is that submit means different things depending on where the CheckList is (Input or Build).